crates/libs/core/src/imp/mod.rs

```diff
@@ -10,6 +10,7 @@ mod sha1;
 mod waiter;
 mod weak_ref_count;
 
+pub use crate::com_object::{ComImpl, ComObjectInterface};
```

Comment:
Nit: none of the other `use` declarations prefix with `crate::`

Reply:
Removed and used crate root.